app/addons/replication/actions.js

```diff
@@ -231,15 +231,15 @@ const hideConflictModal = () => {
   });
 };
 
-const updateUsernameAndPassword = (username, password) => {
-  FauxtonAPI.dispatch({
-    type: ActionTypes.REPLICATION_USERNAME_PASSWORD,
-    options: {
-      username,
-      password
-    }
-  });
-};
+// const updateUsernameAndPassword = (username, password) => {
+//   FauxtonAPI.dispatch({
+//     type: ActionTypes.REPLICATION_USERNAME_PASSWORD,
+//     options: {
+//       username,
+//       password
+//     }
+//   });
+// };
```

Comment:
I'd recommend deleting this versus leaving it commented.  Seems like other unused functions were deleted elsewhere in the PR.